### DISCUSSION TITLE
Configure roles - does not appear

### DISCUSSION THREAD

**Discussion by GuilhermeHirsch** [Budibase/budibase#2194] [Help] [answered]
Guys, I noticed a strange behavior with the configure roles. I created a fourth app, but no user settings appeared. I did this test by creating a new app and importing an already functional one. It looks like only my first three apps are visible. 

![Sem título](https://user-images.githubusercontent.com/33108596/127652720-01d40dfa-948b-4010-8c0f-7df77687ecfb.jpg)

![Sem título2](https://user-images.githubusercontent.com/33108596/127652735-b8691431-56d5-48cc-961d-78a64a33d520.jpg)

Another question: Is it possible to create a custom role to be accessed outside the app?

**Comment by mjashanks** [comment, marked as answer]
Hey @GuilhermeHirsch - I can reproduce your issue also. It seems to only affect recently created apps. I'm not sure how recent. Will look into this.

> Is it possible to create a custom role to be accessed outside the app?

You can create a new role within your app:
1. Click on the Users table
2. Click edit roles

Your new role should now appear when assigning to a user (as your screenshot above). Make sure you are on `0.9.86` as we had a bug around this in `0.9.85`

**Reply by GuilhermeHirsch** [reply]
Alright, @mjashanks. I managed to create a role and access. Yes, it's in version 0.9.86.

**Reply by mjashanks** [reply]
I just noticed that after I updated my own Budibase, my new apps appeared. So, a workaround for now is to restart your services. I use the budibase cli, so for me its 
```
budi hosting --stop
budi hosting --start
```

**Reply by GuilhermeHirsch** [reply]
Wow! It worked here, @mjashanks! Thank you! Have a great day!
